pegasus/sites.v3/code.org/public/css/page/donate-styles-draft.scss

```diff
@@ -0,0 +1,86 @@
+// Custom styles for code.org/donate
+// — supplemental to shared/css/design-system-pegasus.scss
+
+@import "breakpoints";
+
+section.top {
+  .flex-container {
+    gap: 1rem;
+  }
+}
+
+section.donate-options {
+  @media screen and (max-width: $width-md) {
+    .wrapper > hr {
+      display: none;
+    }
+  }
+
+  .tabs-section {
+    min-height: 307px;
+    margin-bottom: 3rem;
+
+    button.tab {
+      flex: 1;
+    }
+
+    @media screen and (max-width: $width-md) {
+      margin-bottom: 2rem;
+    }
+
+    #dafdirectdiv {
```

Comment:
What does this name stand for (what is 'daf') and why do they need the !important tag?

Reply:
It's this, and I'm overwriting styles coming from their embed. 

<img width="1031" alt="Screenshot 2024-08-02 at 9 31 54 AM" src="https://github.com/user-attachments/assets/7a74362c-e228-4751-9ad7-98ca7b004f10">

Reply:
ahhh yes yes. Thanks!